tasks/main.yml

```diff
@@ -44,3 +44,21 @@
     state: present
     regexp: '^%PWM_APPLICATIONPATH'
     line: 'PWM_APPLICATIONPATH={{ pwm_path_config }}'
+
+- name: Copy themes
+  synchronize:
+    src: 'host_vars/{{ inventory_hostname }}/files/themes/{{ item }}'
+    dest: /var/lib/tomcat/webapps/pwm/public/resources/themes
+    delete: no
+    recursive: yes
+  with_items: '{{ pwm_interface_theme_add }}'
+  when: pwm_interface_theme_add is defined
+
+- name: Change themes, group and permissions
+  file:
+    path: /var/lib/tomcat/webapps/pwm/public/resources/themes
```

Comment:
there is no var for tomcat installation directory?

maybe it would be better to make one. `pwm_tomcat_location` or something like that

Reply:
Brilliant!! I add two variables one for tomcat location (pwm_path_tomcat) and one for themes (pwm_path_themes)